ui/app/components/app/whats-new-popup/whats-new-popup.js

```diff
@@ -0,0 +1,177 @@
+import React, { useContext, useMemo, useRef, useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { I18nContext } from '../../../contexts/i18n';
+import { useEqualityCheck } from '../../../hooks/useEqualityCheck';
+import Button from '../../ui/button';
+import Popover from '../../ui/popover';
+import { updateViewedNotifications } from '../../../store/actions';
+import { getTranslatedUINoficiations } from '../../../../../shared/notifications';
+import { getSortedNotificationsToShow } from '../../../selectors';
+
+function getActionFunctionById(id) {
+  const actionFunctions = {
+    2: () => {
+      global.platform.openTab({
+        url:
+          'https://survey.alchemer.com/s3/6173069/MetaMask-Extension-NPS-January-2021',
+      });
+    },
+    3: () => {
+      global.platform.openTab({
+        url: 'https://community.metamask.io/t/about-the-security-category/72',
+      });
+    },
+  };
+
+  return actionFunctions[id];
+}
+
+const renderFirstNotification = (notification, idRefMap) => {
+  const { id, date, title, description, image, actionText } = notification;
+  const actionFunction = getActionFunctionById(id);
+  return (
+    <div
+      className={classnames(
+        'whats-new-popup__notification whats-new-popup__first-notification',
+      )}
+      key={`whats-new-popop-notificatiion-${id}`}
+      ref={idRefMap[id]}
+    >
+      {image && (
+        <img
+          className="whats-new-popup__notification-image"
+          src={image.src}
+          height={image.height}
+          width={image.width}
+        />
+      )}
+      <div className="whats-new-popup__notification-title">{title}</div>
+      <div className="whats-new-popup__description-and-date">
+        <div className="whats-new-popup__notification-description">
+          {description}
+        </div>
+        <div className="whats-new-popup__notification-date">{date}</div>
```

Comment:
Suggestion: Rather than displaying ISO-8601 formatted dates, we should format them according to the user's current locale using something like `new Intl.DateTimeFormat(locale).format(date)`. And we'd want to do the same for the subsequent notifications as well of course.

[This API is supported in all browsers we support](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat), and generally we should display all numbers, dates, and times as localized wherever possible.

Reply:
@Gudahtt Dates updated (according to spec from Jacob) and localized here: 7b120a4af